packages/common/aws.js

```diff
@@ -146,47 +146,15 @@ exports.downloadS3File = (s3Obj, filename) => {
   });
 };
 
-/**
-* Put an object on S3
-* @param {object} options
-* @param {object} options.body
-* @param {object} options.bucket name of bucket
-* @param {object} options.key key for object (filepath + filename)
-* @param {object} options.body body of the file
-* @param {string} [options.acl=private] body of the file
-* @param {object} [options.Metadata=null] body of the file
-* @returns {promise} returns response from `S3.putObject` as a promise
-**/
-exports.putS3Object = (options) => {
```

Comment:
Didn't get a chance to check but are we sure this function is not used by any of the tasks or libraries?

Reply:
It isn't used anywhere.